CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# v1.8.0:
+
+### Features:
+
+- [Feature #35](https://github.com/google/osv-scanner/issues/35)
```

Comment:
This issue hasn't been closed - should it be?

Reply:
I will close it once the new version is released.

Reply:
I think so, @cuixq?